common/ast/src/ast/statements/share.rs

```diff
@@ -96,3 +96,37 @@ impl Display for RevokeShareObjectStmt<'_> {
         Ok(())
     }
 }
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct AlterShareAccountsStmt<'a> {
+    pub share: Identifier<'a>,
+    pub if_exists: bool,
+    pub tenants: Vec<Identifier<'a>>,
+    pub add: bool,
```

Comment:
How about changing into `is_add`? Or, we can use an `enum` here which makes it easier for parser.